backend/src/openbeheer/zaaktype/api/views.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+from typing import Mapping
+
+from msgspec import UNSET, Struct, UnsetType, field
+from openbeheer.api.views import ListView
+from openbeheer.types.ztc import Status, VertrouwelijkheidaanduidingEnum
+from openbeheer.types import OBPagedQueryParams, OBField, OBOption
+
+
+class ZaaktypenGetParametersQuery(OBPagedQueryParams, kw_only=True):
+    catalogus: int | None = None
+    datum_geldigheid: str | None = field(name="datumGeldigheid", default=None)
+    identificatie: str | None = None
+    page: int = 1
+    status: Status | None = None
+    trefwoorden: list[str] = []
+
+
+class ZaakType(Struct, kw_only=True):
+    # Identificate
+    # Omschrijving
+    # Actief ja/nee
+    # Einddatum
+    # Omschrijving
+    # Vertrouwelijkheidaanduiding
+
+    identificatie: str
```

Comment:
Came across this when working on the detail page. I think that we will need uuid/url also, because otherwise we cannot retrieve the zaaktype (no other field guarantees to be unique)